extensions/amp-form/0.1/form-validators.js

```diff
@@ -323,18 +305,16 @@ export class AbstractCustomValidator extends FormValidator {
   }
 }
 
-
 /** @private visible for testing */
 export class ShowFirstOnSubmitValidator extends AbstractCustomValidator {
-
   /** @override */
   report() {
     this.hideAllValidations();
     const inputs = this.inputs();
     for (let i = 0; i < inputs.length; i++) {
       if (!inputs[i].checkValidity()) {
         this.reportInput(inputs[i]);
-        inputs[i]./*REVIEW*/focus();
+        inputs[i]./*REVIEW*/ focus();
```

Comment:
@yomeshgupta Could you please revert the unrelated formatting changes? Maybe your IDE automatically added them.